src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -103,27 +118,43 @@ function BaseHeader(props: HeaderProps) {
   const icons: { [key: string]: JSX.Element } = {
     backNav: getIcon(backAction || backActionDefault, <ArrowBackIcon />),
     edit: getIcon(navigateToEdit, <CreateIcon />, true),
-    user: getIcon(openProfileMenu, <AccountCircleIcon className={classes.account} fontSize="large" />),
+    user: getIcon(openProfileMenu, <MenuIcon className={classes.account} fontSize="large" />),
   };
 
   const left = leftIcon ? icons[leftIcon] : null;
   const header = title ? (
-    <Typography className={classes.title} variant="h2">
+    <Typography className={searchAction ? classes.leftTitle : classes.title} variant="h2">
       {title}
     </Typography>
   ) : null;
   const right = rightIcon ? icons[rightIcon] : null;
 
+  const onSearchExit = () => {
+    setSearchVisibility(false);
+    searchExit();
+  }
+
   return (
-    <div className={classes.root}>
+    <div className={classes.root} style={{ marginTop: props.searchAction ? '20px' : undefined }}>
       {header}
       <div className={classes.toolbar}>
         <div className={classes.left}>{left}</div>
-        <div className={classes.right}>{right}</div>
+        <div className={classes.right}>
+          {props.searchAction ? getIcon(() => setSearchVisibility(true), <SearchIcon className={classes.search} />) : null}
+          {right}
+        </div>
         {profileMenu}
       </div>
+      <div className={classes.searchBar} style={{display: searchVisibility ? 'block' : 'none' }}>
+        <SearchBar placeholder="Search for a customer" onSearchChange={searchAction} onSearchExit={onSearchExit} />
+      </div>
     </div>
   );
 }
 
-export default withStyles(styles)(BaseHeader);
+const mapStateToProps = (state: RootState) => ({
+  name: state.userData.user?.fields.Name || '',
+  email: state.userData.user?.fields.Email || '',
+});
```

Comment:
Remove this and replace with the selectors that got deleted -- this is breaking it
![image](https://user-images.githubusercontent.com/21160510/114430377-38b71900-9b73-11eb-87f5-0a9838a2bc93.png)